### PR TITLE
doc(examples): add an example for using server data (without macros)

### PR DESCRIPTION
**Pull request by lipanski**
Related to my question [here](https://github.com/nickel-org/nickel.rs/issues/351).

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -126,3 +126,8 @@ path = "examples/form_data/form_data.rs"
 
 name = "https"
 path = "examples/https.rs"
+
+[[example]]
+
+name = "server_with_data"
+path = "examples/server_with_data.rs"
```
Comment (Ryman) [inline Cargo.toml]: Could this be renamed to something that makes it clear that there's no macro use? Perhaps your branch name `no_macro_custom_data`?

**Comment by lipanski** [comment]
@Ryman updated!

**Comment by Ryman** [comment]
@homu r+

**Comment by Ryman** [comment]
Thanks @lipanski !

**Comment by lipanski** [comment]
:balloon:

**Pull request merged**